test/integration/targets/nxos_bfd_interfaces/tests/cli/overridden.yaml

```diff
@@ -0,0 +1,65 @@
+---
+- debug:
+    msg: "Start nxos_bfd_interfaces overridden integration tests connection={{ ansible_connection }}"
+
+- set_fact: test_int1="{{ nxos_int1 }}"
+- set_fact:
```

Comment:
Please add one more interface in tests for overridden, so we can distinguish between overridden and replaced.